src/guides/libraries/c-interop.md

```diff
@@ -253,6 +253,65 @@ To reduce this burden,
 you can use the [`package:ffigen`][ffigen] binding generator
 to automatically create FFI wrappers from C header files.
 
+<a id="native-assets"></a>
+## Building and bundling native assets
+
+{{site.alert.note}}
+  The native assets feature is **experimental**,
+  and [in active development](https://github.com/dart-lang/sdk/issues/50565).
+{{site.alert.end}}
+
+The Native Assets feature aims to resolve a number of issues associated with
+the distribution of Dart packages that depend on native code.
+It does so by providing uniform hooks for integrating with various
+build systems involved in building Flutter and standalone Dart applications.
+
+The Native Assets feature aims to make it seamless for
+Dart packages to depend on and use native code:
+
+* It builds (if needed) the native code or
+  obtains the binaries using a package's `build.dart` script.
+* It bundles the native [`Asset`][] reported by the `build.dart` script.
+* It makes the native assets available at runtime through
+  declarative `@Native<>() extern` functions using the [`assetId`][].
+
+The `flutter run` / `flutter build` and `dart` / `dart build` tools
+will now build and bundle native code when
+[opted in](#experiment-opt-in) to the native experiment.
```

Comment:
Should this be specific about the flag used?

```suggestion
[opted in](#experiment-opt-in) to the `native-assets` experiment.
```

Reply:
The info on how to opt in is in the two linked issues. It's a it different for Dart than for Flutter. Flutter is a global persistent config, Dart is a per-invocation command-line flag.

We could add it at the bottom if you prefer. Martin suggested the same earlier.

Reply:
Sorry I sometimes think too much on the Dart side of things. I think what you have now makes sense!

Could you update your two issues to have more a prominent "Try now" h2 or h3 header or similar with the instructions. The small notes at the top may be a bit easy for readers to miss.

Reply:
Updated the issues to be bold at the top, fixed the Dart command (the experiment flag should be _before_ `run`) and updated the message to refer to the stable versions.